integration/init_test.go

```diff
@@ -52,18 +57,17 @@ func TestInitCompose(t *testing.T) {
 
 			initArgs := append([]string{"--force"}, test.args...)
 			skaffold.Init(initArgs...).InDir(test.dir).WithConfig("skaffold.yaml.out").RunOrFail(t.T)
-
 			checkGeneratedConfig(t, test.dir)
 
 			// Make sure the skaffold yaml and the kubernetes manifests created by kompose are ok
```

Comment:
nit: this comment doesn't make sense anymore as this test is no longer just for compose/kompose testing but also for helm